internal/pkg/agent/install/install.go

```diff
@@ -109,35 +110,30 @@ func Install(cfgFile, topPath string, unprivileged bool, log *logp.Logger, pt *p
 			errors.M("directory", filepath.Dir(topPath)))
 	}
 
-	// copy source into install path
-	//
-	// Try to detect if we are running with SSDs. If we are increase the copy concurrency,
-	// otherwise fall back to the default.
-	copyConcurrency := 1
-	hasSSDs, detectHWErr := HasAllSSDs()
-	if detectHWErr != nil {
-		fmt.Fprintf(streams.Out, "Could not determine block hardware type, disabling copy concurrency: %s\n", detectHWErr)
+	manifestFile, err := os.Open("manifest.yaml")
```

Comment:
can we extract work with manifest into separate function and release handler a bit sooner?

Reply:
done in acacac7efb7732c819ab80fe334706b841775cab